plugins/tech-radar/src/components/RadarDescription/RadarDescription.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import Dialog from '@material-ui/core/Dialog';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import { Button, DialogActions, DialogContent } from '@material-ui/core';
+import LinkIcon from '@material-ui/icons/Link';
+
+export type Props = {
+  open: boolean;
+  onClose: () => void;
+  title: string;
+  description: string;
+  url?: string;
+};
+
+const RadarDescription = (props: Props): JSX.Element => {
+  // const classes = useStyles(props);
+  const { open, onClose, title, description, url } = props;
+
+  const handleClick = () => {
+    onClose();
+    if (url) {
+      window.location.href = url;
+    }
+  };
+
+  return (
+    <Dialog data-testid="radar-description" open={open} onClose={onClose}>
+      <DialogTitle id="radar-description-dialog-title">{title}</DialogTitle>
+      <DialogContent dividers>{description}</DialogContent>
+      {url && (
+        <DialogActions>
+          <Button
+            onClick={handleClick}
+            color="primary"
+            startIcon={<LinkIcon />}
+            variant="contained"
+          >
+            Visit URL
```

Comment:
oops lost my comment!

Suggest this be "Learn More". To an end user, they aren't WANTING to "visit a url", they would want to click the button to find more out about this tech radar suggestion.

Trying to align to some good suggestions here https://material.io/components/buttons#text-button which I've recently been made aware the maintainers are intending to continue to align to (in alignment with the overall use of the Material-UI react library).

Reply:
makes more sense I'll update.